src/core/libmaven/csvreports.cpp

```diff
@@ -359,6 +364,12 @@ void CSVReports::writePeakInfo(PeakGroup* group) {
         compoundName = sanitizeString(group->compound->name.c_str()).toStdString();
         compoundID   = sanitizeString(group->compound->id.c_str()).toStdString();
         formula = sanitizeString(group->compound->formula.c_str()).toStdString();
+    } else {
+        // absence of a group compound means this group was created using untargeted detection,
+        // we set compound name and ID to {mz}@{rt} strings for untargeted sets.
+        cout << "here" << endl;
```

Comment:
Not needed I guess

Reply:
My bad. Was using this for debugging. Will remove